adapter/repository/in-memory-simple/src/main/java/org/mobilitydata/gtfsvalidator/db/InMemoryExecParamRepository.java

```diff
@@ -235,9 +235,12 @@ public String getExecParamValue(final String key) throws IllegalArgumentExceptio
             }
 
             case EXCLUSION_KEY: {
-                return hasExecParamValue(EXCLUSION_KEY) ?
-                        getExecParamByKey(EXCLUSION_KEY).getValue().toString()
-                        : null;
+                return hasExecParamValue(EXCLUSION_KEY) ? getExecParamByKey(EXCLUSION_KEY).getValue().toString() : null;
+            }
+
+            case TRANSFER_MIN_TRANSFER_TIME_RANGE_MIN :
```

Comment:
Doesn't look like you've added the rule description yet, but I'd definitely include the default range values in the rule description, as well as a message saying they can be configured an a hyperlink to the documentation describing how ranges can be configured.